python/whylogs/api/logger/experimental/logger/actor/process_rolling_logger.py

```diff
@@ -93,7 +93,7 @@ def create_writers(self, dataset_id: str) -> List[Writer]:
 class WhyLabsWriterFactory(WriterFactory):
     def create_writers(self, dataset_id: str) -> List[Writer]:
         return [
-            Writers.get(
+            Writers.get(  # type: ignore
```

Comment:
```suggestion
            Writers.get(  # type: ignore E405
```

Reply:
It doesn't like that ignore syntax

Reply:
flake8 uses `# noqa: E{pep8 error you chose to ignore}`